src/ParaTest/Runners/PHPUnit/SuiteLoader.php

```diff
@@ -230,7 +236,7 @@ private function getMethodTests($class, $method, $useDataProvider = false)
 
     private function testMatchGroupOptions($groups)
     {
-        if (empty($groups)) {
+        if (empty($this->options->groups) && empty($this->options->excludeGroups) && empty($groups)) {
```

Comment:
this PR needs work anyways. So either there are tests added and your issue is fixed @carlo1138 or this PR won't get merged.